ground/src/main/java/com/google/android/ground/ui/editsubmission/EditSubmissionFragment.java

```diff
@@ -16,13 +16,8 @@
 
 package com.google.android.ground.ui.editsubmission;
```

Comment:
Is there anything left in this file that we need? Or do you think it can be removed now?

Reply:
I'll clean it up in a follow up PR.